packer-scripts/run-serverspecs

```diff
@@ -48,8 +48,8 @@ __install_chefdk() {
 
   apt-key add "${tmp}/chef.asc"
 
-  echo "deb https://packages.chef.io/current-apt $(lsb_release -cs) main" \
```

Comment:
IIUC there are `current` and `stable` channels available in the chef APT repo, but sometime between last Friday and today the `current` repo stopped publishing the `chefdk` package.  I didn't investigate into why this was the case, since the `stable` repo works fine for my purposes.